BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -25,6 +24,15 @@ class AutoContributeDetailViewController: UIViewController {
     setupLedgerObservers()
   }
   
+  func setupLedgerObservers() {
+    ledgerObserver.excludedSitesChanged = { _, _ in
```

Comment:
this retains cycle, add `weak self` here or make ledgerObserver weak

Reply:
Good catch! 😃